src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -160,6 +165,10 @@ private AxeView(
     this.viewIdResourceName = viewIdResourceName;
     this.children = children;
 
+    if (this.viewIdResourceName != null && this.viewIdResourceName.contains("content")) {
```

Comment:
The second check in here contains a failure state. A user could have a view named `"@+id/blog_post_content"`, or somesuch, and the contentViewAxeRect will be set to the incorrect boundsInScreen.

Grab a few devices and an emulator, check that the content view has the same content resource name, then fully qualify the string comparison here. It should look something like `android.R.foo.bar:id/content`

Reply:
After debugging I found it to be `android:id/content`